src/main/java/org/jfree/chart/date/SpreadsheetDate.java

```diff
@@ -84,32 +87,20 @@
      * @param year  the year (in the range 1900 to 9999).
```

Comment:
Delete the text in parenthesis in the javadoc for year parameter

Reply:
OK done